static/js/app.js

```diff
@@ -239,10 +246,22 @@ function addEventListeners() {
       }
 
       sendGaPing("Generate New Alias", "Engage", "btn-enabled");
+    
       e.target.submit();
+      localStorage.setItem("generateBtn", "True");
```

Comment:
Hmm, I don't understand why we need this? (Does the backend set something like this? Since it's capitalised like Python's `True`?)

Reply:
I used `localStorage` to check that the "Generate an Alias" button was selected even when the site refreshes when an alias is generated.

Reply:
(so upon using React... we probably don't need any more than a setting an attribute)

Reply:
Hmm, but isn't the top alias _always_ supposed to be expanded? (Which will be the just-generated alias if you refresh the page.)

Reply:
Ah gotcha, that's good to know so I know what to test :) (And also because the React implementation currently works differently, so I'll also send a PR for that :) )